config/services.php

```diff
@@ -9,4 +9,16 @@
         'scheme' => 'https',
     ],
 
+    'github' => [
+        'client_id' => env('OAUTH_GITHUB_CLIENT_ID'),
+        'client_secret' => env('OAUTH_GITHUB_CLIENT_SECRET'),
+        'redirect' => '/auth/oauth/callback/github',
+    ],
+
```

Comment:
```suggestion
```

Don't think we need this as a default, plugin probably.

Reply:
It's included by default in socialite and nice for testing. That's why I added it.

Reply:
Ah okay, sounds good then!